src/main/java/uk/gov/hmcts/reform/sscs/config/AppConstants.java

```diff
@@ -45,6 +45,7 @@
     public static final String ONLINE_HEARING_SIGN_IN_LINK_LITERAL = "online_hearing_sign_in_link";
     public static final String PANEL_COMPOSITION = "panel_composition";
     public static final String PHONE_NUMBER = "phone_number";
+    public static final String HMCTS_PHONE_NUMBER = "hmcts_phone_number";
```

Comment:
Why have we added this new field? It's going to break every template for Reps when this is merged?